invisible_cities/detsim/buffer_functions.py

```diff
@@ -141,6 +141,9 @@ def position_signal(triggers   :       List                  ,
             pmt_q  = np.asarray(pmt_charge.tolist())
             sipm_q = np.empty((0,0))\
             if sipm_charge.empty else np.asarray(sipm_charge.tolist())
+        else:
```

Comment:
Could you add a test for this, it should be very straightforward. Something similar to 
`test_buffer_calculator` flow but just asserting that the output is the same when using pandas and numpy array as input?